apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/EndpointJsonResponseComparatorIT.java

```diff
@@ -201,16 +201,14 @@ public void verifyCorrectEndpointResponse(String endpointId, Supplier<String> en
   }
 
   /**
-   * Generates the "golden" files, i.e. the approved responses to compare to. Purpose of this
-   * testing is to perform regression testing against the "Golden Beneficiary Data" at a specific
-   * point in time. It is important to note that this testing focuses on checking for regressions
-   * against the data at that particular moment, and not necessarily against data artifacts. To run
-   * this test, execute the following Maven Command: mvn clean install -DgenerateTestData=true
+   * Generates the "golden" files, i.e. the approved responses to compare to. Run by commenting out
+   * the <code>@Disabled</code> annotation and running this method as JUnit.
    *
    * @param endpointId the endpoint id
    * @param endpointOperation the endpoint operation
    */
-  @EnabledIfSystemProperty(named = "generateTestData", matches = "true")
+  @Disabled(
+      "This should not be a test, Should be handled in another location as part of build flow")
```

Comment:
+1 to Brian's comment about this undoing a change for both the `EndpointJsonResponseComparatorIT.java` and `EndpointJsonResponseComparatorV2IT.java`